srcpkgs/webkit2gtk/template

```diff
@@ -132,21 +134,132 @@ pre_configure() {
 	esac
 }
 
+post_configure() {
+	mkdir -p webkit2gtk-41
+	pushd webkit2gtk-41
+		configure_args=${configure_args//-DUSE_SOUP2=ON/}
+		configure_args=${configure_args//-DENABLE_WEBDRIVER=OFF/}
+		do_configure
+	popd
+	mkdir -p webkit2gtk-50
+	pushd webkit2gtk-50
+		configure_args+=" -DUSE_GTK4=ON -DENABLE_WEBDRIVER=OFF"
+		do_configure
+	popd
```

Comment:
```suggestion
	mkdir -p webkit2gtk-40 webkit2gtk-50
	(
		cd webkit2gtk-40 && \
		configure_args="${configure_args} -DUSE_SOUP2=ON -DENABLE_WEBDRIVER=OFF" \
		do_configure
	)
	
	(
		cd webkit2gtk-50 && \
		configure_args="${configure_args} -DUSE_GTK4=ON -DENABLE_WEBDRIVER=OFF" \
		do_configure
	)
```
Temporarily manipulating state like working directory or the value of a variable is undesirable and prone to error. Use subshells instead.

Also, because I modified the default args, it seems the default `do_build` run will produce `webkit2gtk41`, so I've swapped `webkit2gtk-41` for `webkit2gtk-40` here. You'll have to double-check my work.

Reply:
Thanks for the tip.